docs/news.d/openddsconfig.rst

```diff
@@ -0,0 +1,7 @@
+.. news-prs: 4482
+
+.. news-start-section: Additions
+- A new header, dds/OpenDDSConfig.h is generated by configure or CMake.
+  - Users manually configuring a build will need to create this file, which may be empty.
+  - See dds/OpenDDSConfig.h.in for details.
```

Comment:
```suggestion
- A new header, ``dds/OpenDDSConfig.h`` is generated by configure or CMake.

  - Users manually configuring a build will need to create this file, which may be empty.
  - See ``dds/OpenDDSConfig.h.in`` for details.
```

Reply:
Isn't it possible to create the empty file in the repo so that people just can build it (several projects embed OpenDDS build within a larger build environment)

Reply:
Would it be possible to add a ifdef around the include, that way we can add a define to ace/config.h and don’t have to extend our build dsl, very likely others can use this also

Reply:
As far as I understand what you are asking, I'm okay with it.  However, let's do that in another PR.

Reply:
We build and test daily against master, merging this will break that